plugin.php

```diff
@@ -282,6 +282,13 @@ function register_api_field( $object_type, $attributes, $args = array() ) {
 	 * @return WP_Error|boolean
 	 */
 	function rest_validate_request_arg( $value, $request, $param ) {
+		// for validating filter args
+		if ( preg_match( "/filter\[([a-zA-Z0-9_]+)\]/", $param, $m )) {
+			$filters = $request->get_param( 'filter' );
+			if ( ! empty( $filters[ $m[1] ] ) ) {
+				$value = $filters[ $m[1] ];
+			}
+		}
```

Comment:
Not sure this is the best approach and I would also recommend wrapping this in an `if ( isset( $request['filter'] ) )`. No reason to do a regex check for every time this function is called.  Also naming $m to $matches would be preferred in my opinion, as it makes the code more readable.  There is also a missing space on line 286 before the last parenthesis. :smile: